clar2wasm/tests/wasm-generation/noop.rs

```diff
@@ -0,0 +1,34 @@
+use clar2wasm::tools::crosscheck;
+use clarity::vm::Value;
+use proptest::proptest;
+use proptest::strategy::Strategy;
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crosscheck_noop_to_uint(val in (0u128..=126)
+        .prop_map(|exp| 2u128.pow(exp.try_into().unwrap()))
+        .prop_map(Value::UInt)
+    ) {
+        crosscheck(
+            &format!("(to-uint (to-int {val}))"),
+            Ok(Some(val))
```

Comment:
I think the test should be simpler and just try `to-uint`. This is actually what the test name suggests.

I think this test should just generate any `i128`, find the expected result and crosscheck.

Reply:
Ok... changed for a simple implementation.